test/main.sh

```diff
@@ -110,6 +110,11 @@ cleanup() {
     if [ "${expandDmesg}" = "no" ]; then
       echo "::endgroup::"
     fi
+
+    echo "::group::crashes"
```

Comment:
should we make this group conditional based on whether there is anything in /var/crash/?

Reply:
Have done!